docker/production.ini

```diff
@@ -67,6 +73,14 @@ args = (sys.stderr,)
 level = NOTSET
 formatter = generic
 
+[handler_sqlalchemy_logger]
+class = c2cwsgiutils.sqlalchemylogger.handlers.SQLAlchemyHandler
+#args = ({'url':'sqlite:///logger_db.sqlite3','tablename':'test'},'curl')
+args = ({'url':'postgresql://postgres:password@oereb-db:5432/stats','tablename':'logs','tableargs': {'schema':'oereb_logs'}},'healthcheck')
```

Comment:
Please use "oereb_stats" instead of "stats" as database name

Reply:
fixed [here](https://github.com/danduk82/pyramid_oereb/pull/2/commits/39b9c20f9ed19f48e64874f7fb0fb1ad88c036ec#diff-9d3e223fe6044c8c5392a733b202cc08R79)